### PR TITLE
docs: add planned release schedule till v16

### PR DESCRIPTION
**Pull request by thesmiler**
Add release dates for v15.1,v15.2 and v16

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Release dates are only mentioned till v15.0

## What is the new behavior?

Planned release dates till v16.0 are added

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

Reviewed-for: global-approvers

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -112,7 +112,9 @@ Dates are offered as general guidance and are subject to change.
 
 | Version | Date               |
 | :---    | :---               |
-| v15.0   | Week of 2022-11-18 |
+| v15.1   | Week of 2022-01-09 |
```
Comment (tonivj5) [inline aio/content/guide/releases.md:115]:
```suggestion
| v15.1   | Week of 2023-01-09 |
```
this and the rest 😄

Comment (thesmiler) [reply aio/content/guide/releases.md]: Thanks! Good catch!

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit b10e6185de7efec5dc5664631712cc6e571069c7.

**Pull request closed without merging**